modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/reader/BatchEngineImportTaskItemReaderUtil.java

```diff
@@ -69,7 +92,30 @@ public static <T> T convertValue(
 			if (field != null) {
 				field.setAccessible(true);
 
-				ObjectMapper objectMapper = _getObjectMapper(field);
+				ObjectMapper objectMapper = null;
+
```

Comment:
As we talked, maybe there is a different way, by using `Deserializer` or `BeanSerializerModifier`. Can you try to explore it?

Reply:
Hi @ccorreagg ,

I've checked the `simpleModule.setDeserializerModifier(..)`, however when the methods of this class are called, the configuration of the deserializer is already decided. For example, the `externalReferenceCode` property doesn't even have a setter method defined in the property config when it's READ_ONLY.
My other idea is to still use the Mixin, but get rid of the dependency on the `com.liferay.headless.delivery.dto.v1_0.Creator` class. See example: https://github.com/vendeltoreki/liferay-portal/commit/63ca8d6ff1ce80b0a27056b83309e1ca5efade38